CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+### v1.1.0 / 2024-10-22
```

Comment:
After previous release, we discussed not adding dates in files anymore, to avoid mismatches when package is being merged with upstream. It's easy to overlook changing date here before merge.

...so, this is an open question. Are we committed to not adding dates in files anymore?

Reply:
This is fine to me!